webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -1169,6 +1171,61 @@ msgstr ""
 msgid "form.lotse.confirm_complete_correct.required"
 msgstr "Bestätigen Sie, dass Ihre Angaben korrekt sind, um fortfahren zu können."
 
+#: app/forms/steps/lotse/fahrkostenpauschbetrag.py:33
+msgid "currency.euro"
+msgstr "Euro"
```

Comment:
Shouldn't this be used in more places than just once?

Reply:
It is used here and in the pauschbetrag step. However, this has not been merged.